uReplicator-Manager/src/main/java/com/uber/stream/kafka/mirrormaker/manager/core/ControllerHelixManager.java

```diff
@@ -117,14 +122,10 @@
 
   private ZkClient _zkClient;
 
-  private boolean _enableAutoScaling = true;
-  private boolean _enableRebalance;
-
   public ControllerHelixManager(
       KafkaClusterValidationManager kafkaValidationManager,
       ManagerConf managerConf) {
     _conf = managerConf;
-    _enableRebalance = managerConf.getEnableRebalance();
```

Comment:
shall we keep the `managerConf.getEnableRebalance();` so users can specify the default value during service startup?

Reply:
sure